python/tvm/relay/frontend/tflite.py

```diff
@@ -1634,6 +1636,56 @@ def convert_reduce_sum(self, op):
     def convert_reduce_any(self, op):
         return self._convert_reduce(_op.reduce.any, op)
 
+    def _convert_arg_min_max(self, relay_op, op):
+        """Generic method to convert TFLite arg_min_max"""
+        try:
+            from tflite.Operator import Operator
```

Comment:
Remove this. not required

Reply:
Done

Reply:
@d-smirnov 

is something like this?

    def convert_arg_max(self, op):
        """Convert TFLite ABS"""
        if self.is_quantized(op):
            raise tvm.error.OpNotImplemented(
                'TFlite quantized ARG_MAX operator is not supported yet.')
        return self._convert_unary_elemwise(_op.arg_max, op)